README.md

```diff
@@ -0,0 +1,40 @@
+# Lansing Codes website
+
+[Lansing Codes](https://www.lansing.codes) is a website of resources for coders
+and other tech enthusiasts in and around Lansing, Michigan.
+
+These resources include a calendar of tech events, a list of meetup groups,
+local beginner-friendly resources, shout-outs of our gracious sponsors, and
+other ways to get involved in the Lansing tech community.
```

Comment:
Do you mind updating the title and description to the repository name and description? I think this will do:

``` md
# Lansing Codes Newsletter Events

Shows next month's events in a newsletter-friendly list.
You can view the list in your browser at
[lansingcodes.github.io/newsletter-events](https://lansingcodes.github.io/newsletter-events/).

Reply:
Sure, sorry i didnt pay close attention to the title of PR. Will make sure it has repository name.

Reply:
Cool, @ me on GitHub or Slack when it's ready and I'll take another look. 😎

Reply:
Thank you ..

Reply:
It looks like the heading and description at the top of `README.md` are still referring to the Lansing Codes website. Are you still working on that change?